.bazelrc

```diff
@@ -1,5 +1,5 @@
 startup --expand_configs_in_place
-common --incompatible_disallow_filetype=false
+common --incompatible_bzl_disallow_load_after_statement=false
```

Comment:
I am using the same Bazel version but I don't need this flag in `rules_play_routes`. If it's necessary here, just keep it.

Reply:
I thought it was needed because of the `WORKSPACE` file before, but it turns out the restriction doesn't apply there. One of the external dependencies used by `rules_twirl` had `load` statements interspersed through a `.bzl` file; updating that dependency to a newer version fixed it